<launch>
    <arg name="given_robot_num" default="2"/>
    <group ns="multi_planning_server">
        <param name="robot_num" value="0"/>
        <param name="given_robot_num" value="$(arg given_robot_num)" />
        <node pkg="multi_planning_server" type="Frontier_Search" name="Frontier_Search"/>
        <!-- <node pkg="multi_planning_server" type="server_planning" name="server_planning"/> -->
        <node pkg="multi_planning_server" type="server_planning" name="server_planning" output="screen"/>
        <!--<node pkg="multi_planning_server" type="costmap_publisher" name="costmap_publisher"/>-->
        <node pkg="multi_planning_server" type="voronoi_updater" name="voronoi_updater" launch-prefix="xterm -e"/>
        <include file="$(find robot_action_mng)/launch/robot_action_mng.launch"/>
    </group>
</launch>